backend/settings.py

```diff
@@ -202,12 +202,48 @@
 }
 
 TINYMCE_DEFAULT_CONFIG = {
-        'menubar': True,
-        "plugins": "advlist,autolink,code,lists,link,anchor,insertdatetime,media,table,paste,wordcount",
-        "toolbar": "formatselect | bold italic backcolor | "
-            "bullist numlist outdent indent | removeformat | code | help",
-        "default_link_target": "_blank",
-    }
+    'menubar': True,
+    "plugins": "advlist,autolink,code,lists,link,anchor,insertdatetime,media,table,paste,wordcount",
+    "toolbar": "bold italic backcolor | "
+        "bullist numlist outdent indent | removeformat | code | help",
+    "default_link_target": "_blank",
+    "invalid_elements": "h1, h2, h3, h4, h5, h6",
+    "invalid_styles": "color, font-family, background-color",
```

Comment:
I think we may want to add `line-height` to this list. The indentation thing I'm not so worried about, but weird line spacing could interfere with easy it is to tell the sections apart.

Reply:
I have added ```line-height``` to ```invalid_styles```, and the line height attributes get filtered out when I pasted the following html text into a text box in a card:   
```
<p style="line-height:10cm">10cm line height</br>
how does this look?</p>
<p style="line-height:80%">80% normal line height</br>
how does this look?</p>
```